tests/integration/utils.ts

```diff
@@ -26,30 +26,50 @@ import type {
 } from '@kiltprotocol/types'
 import { Crypto } from '@kiltprotocol/utils'
 
-import { makeSigningKeyTool } from '../testUtils/index.js'
+import { makeSigningKeyTool } from '../testUtils/TestUtils.js'
```

Comment:
This should be enough since we have an index.ts or not? 

```suggestion
import { makeSigningKeyTool } from '../testUtils'
```

Reply:
A) No, because of ESM - ESM imports need to be fully qualified with extension
B) Due to unrelated issues, importing from the index breaks EVERYTHING

Reply:
(https://github.com/KILTprotocol/ticket/issues/2892)

Reply:
Ah, okay nice 2 know